ADAL/src/ui/ADWebAuthController.m

```diff
@@ -368,7 +385,15 @@ - (void)webAuthDidCancel
 // Authentication completed at the end URL
 - (void)webAuthDidCompleteWithURL:(NSURL *)endURL
 {
-    AD_LOG_INFO(_requestParams.correlationId, @"-webAuthDidCompleteWithURL: %@", endURL);
+    if ([ADAuthorityUtils isKnownHost:endURL])
+    {
+        AD_LOG_INFO(_requestParams.correlationId, @"-webAuthDidCompleteWithURL: %@", endURL);
```

Comment:
endURL can contain lots of PII, like login hint, we can log only the host

Reply:
fixed